### ISSUE TITLE
No support for Razer Raptor Monitor?

### ISSUE THREAD

**Issue by LandingGitHub** [1kc/razer-macos#371]
Hi everyone!

I am new here. I have a MacBook Pro (M1) with a Razer Raptor monitor. I find this project on GitHub for RGB lights to work. But it doesn't seem to recognize my Monitor at all. Is there a way to make it works, or am I totally screwed?

Thank you!

**Comment by LandingGitHub** [comment]
> Razer MacOS drivers are ported from openrazer (for Linux). As of this moment, openrazer has not added support for the Raptor, but does have a request to add it: openrazer/openrazer#1217
> 
> Support for your monitor will be added after its added to openrazer. You can follow up on the issue opened to add it since there hasn't been any activity there for a while. Many devices don't have support yet, so don't worry, but definitely check on it.

Oh ok! Thank you for the he response. The post is 9month old, so I guess it's not a popular hardware unfortunately :/

**Comment by WrathofthePast** [comment]
You can comment on openrazer/openrazer#1217 to ask what the status is. Just to confirm: "Razer Raptor 27" is the exact monitor you are asking about? Otherwise, you should open a new issue on openrazer asking to add support for it.

**Comment by LandingGitHub** [comment]
> You can comment on openrazer/openrazer#1217 to ask what the status is. Just to confirm: "Razer Raptor 27" is the exact monitor you are asking about? Otherwise, you should open a new issue on openrazer asking to add support for it.

Yes it's this monitor. The only Razer monitor. I will try to do that and see if someone will be able to help. I will gladly test with the hardware.

**Comment by egorka105** [comment]
> You can comment on [openrazer/openrazer#1217](https://github.com/openrazer/openrazer/issues/1217) to ask what the status is. Just to confirm: "Razer Raptor 27" is the exact monitor you are asking about? Otherwise, you should open a new issue on openrazer asking to add support for it.

Hello,  

I've been waiting for an update to support the Raptor 27 for a long time.  
Today, I checked again to see if there was anything new and followed the branch you mentioned. It looks like the necessary driver support has already been released:  
[https://github.com/openrazer/openrazer/pull/1764](https://github.com/openrazer/openrazer/pull/1764)  

Could you please let me know if support for the Raptor 27 monitor is planned?  
Thank you!
